db/knexfile.js

```diff
@@ -0,0 +1,22 @@
+// Update with your config settings.
```

Comment:
I guess you can rename `knexfile.js` to `knexConfig.js` or maybe even to `dbConfig.js`